webapp/src/actions/index.js

```diff
@@ -158,71 +158,35 @@ export function getYourPrsDetails(prList) {
 }
 
 export function getLabelOptions(repo) {
-    return async (dispatch, getState) => {
-        let data;
+    return async () => {
         try {
-            data = await Client.getLabels(repo);
+            const data = await Client.getLabels(repo);
+            return {data};
         } catch (error) {
             return {error};
         }
-
-        const connected = await checkAndHandleNotConnected(data)(dispatch, getState);
-        if (!connected) {
-            return {error: data};
-        }
```

Comment:
I think we still need to do this step to handle the error case where the user is not connected